src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseHelper.cs

```diff
@@ -357,13 +357,20 @@ public static void AddTopLevelSuggestionsForCursorType(IntellisenseData.Intellis
                     }
                 }
 
-                FormulaValue[] parameters = callNode.Args.Children.Where(texlNode => texlNode.Kind != NodeKind.Error).Select(texlNode => texlNode switch
+                FormulaValue[] parameters = callNode.Args.Children.Where(texlNode => NoErrorInTexlNode(texlNode))
+                                                                  .Select(texlNode => texlNode switch
                 {
                     StrLitNode strNode => FormulaValue.New(strNode.Value),
                     NumLitNode numNode => FormulaValue.New(numNode.ActualNumValue),
                     _ => null as FormulaValue
                 }).ToArray();
 
+                if (parameters.Any(p => p == null))
+                {
+                    // If some parameters are not either StrLitNode or NumLitNode, we cannot get suggestions
+                    return;
+                }
```

Comment:
how did we hit this?

Reply:
dynamic intellisense context, when passing "a" + "b" and expecting a constant string, we'll hit line 365 and then arrive here